src/components/Profile/components/Identification/index.jsx

```diff
@@ -63,8 +64,8 @@ const Identification = ({ profile, myProf, isOnline, createSnackbar }) => {
       },
       cancelButton: {
         backgroundColor: 'white',
-        color: gordonColors.primary.blue,
-        border: `1px solid ${gordonColors.primary.blue}`,
+        color: 'var(--mui-palette-primary-main)',
```

Comment:
I think you missed a reference to `gordonColors` on line 76 below.

Reply:
Yep, that's already been changed since this PR, it'll be in the next one